src/main/java/org/nibor/autolink/LinkExtractor.java

```diff
@@ -88,8 +93,9 @@ public Builder emailDomainMustHaveDot(boolean emailDomainMustHaveDot) {
          */
         public LinkExtractor build() {
             UrlScanner urlScanner = linkTypes.contains(LinkType.URL) ? new UrlScanner() : null;
+            WwwUrlScanner wwwScanner = linkTypes.contains(LinkType.URL) ? new WwwUrlScanner() : null;
```

Comment:
Sure thing, but should we not have a third type to handle all URLs?
So we have a ```LinkType.HTTP```, ```LinkType.WWW``` and ```LinkType.URL```, whereas the latter includes to two former.
This will not exactly break backwards compatibility, but just add extra functionality to the same methods